app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/RemoveCoInsuredBottomSheetContent.kt

```diff
@@ -24,14 +21,12 @@ import hedvig.resources.R
 @Composable
 internal fun RemoveCoInsuredBottomSheetContent(
   onRemove: (CoInsured) -> Unit,
-  onDismiss: () -> Unit,
   isLoading: Boolean,
-  errorMessage: String?,
+  errorMessage: String?, // todo: not used??
```

Comment:
Yeah if it was not used before let it be I suppose. You can also remove it if you want. It may be that it had error messages that are not useful for the member to see so we stopped showing it?